composer.json

```diff
@@ -24,11 +24,6 @@
             "SilverStripe\\MimeValidator\\Tests\\": "tests/"
         }
     },
-    "extra": {
-        "branch-alias": {
-            "dev-master": "2.x-dev"
-        }
-    },
```

Comment:
I still don't agree with deleting master branch aliases. Big no from me.

Reply:
I'm really not against the branch aliases. I just don't see value in maintaining them on the "bulk approach" basis.
If you'd like to take this responsibility, you might make PRs for all the modules you're interested in maintaining?

Reply:
Respectfully, if you aren't against it then please don't remove it. It doesn't bring any extra benefit (you still have to manually inspect composer.json changes during merge ups anyway), and removes a quantifiable benefit which is that you can install a module with a specific major (or minor) version in mind rather than blindly asking for `dev-master`

Reply:
At this moment I am simply removing incorrect aliases.
I don't want to add new ones because we don't have conventions about it and I see it to be unnecessary for most of the modules. There's no value in having it for modules where we don't plan to do next major yet.
Where maintainers feel strong about it (like @ScopeyNZ just asked me in the [MFA](https://github.com/silverstripe/silverstripe-mfa/pull/366#issuecomment-549471844) module) I think it's fair to have it. Otherwise, I don't think it makes any sense to maintain aliases everywhere for no reason.

Reply:
I think removing them because you don't think they're needed is a mistake. If you're not planning to do another major release, then why have the branch at all? If it's to ensure userhelp doesn't break, then why update the master branch (in context of a major branch) at all? I have a suspicion that doing all of this has already taken longer than a year's worth of updating branch aliases

Anyway... as you've pointed out it's more likely that you'll be maintaining this from now on than me, so let's resolve this topic for now and keep doing what you're doing.

Reply:
we've already discussed that in the main issue - https://github.com/silverstripe/cwp-recipe-kitchen-sink/issues/46

We're trying to achieve more consistency. I'm removing them because they are wrong and will be replaced by a real branch, so there is no need in the alias anymore.

I'm not saying that your suggestions are wrong, I'm just trying to reduce number of changes I have to on "bulk" mode and I don't think we won't be able to add aliases on case by case basis later where it has actual value.

Reply:
Right - in future it's probably worth putting things like this (that affect all core contributors too) in framework. I no longer follow that repo so don't get notifications from it.